beetsplug/web/static/js/beets.js

```diff
@@ -0,0 +1,562 @@
+/* globals _,$,Backbone,Marionette,Howl */
+(function () {
+    "use strict";
+
+    // Constants
+    var SETTINGS = {
+        API: "http://127.0.0.1:8337",
```

Comment:
@eidoom looks like this here is the problem. This needs changing before merging @matlads, we should be able to use `window.location.origin` here instead.

```suggestion
        API: window.location.origin,
```

Reply:
This fixed the problem for me :) Thank you @jackwilsdon